torchvision/prototype/transforms/functional/_geometry.py

```diff
@@ -42,10 +47,9 @@ def resize_image_tensor(
     max_size: Optional[int] = None,
     antialias: Optional[bool] = None,
 ) -> torch.Tensor:
-    # TODO: use _compute_output_size to enable max_size option
-    max_size  # ununsed right now
-    new_height, new_width = size
     num_channels, old_height, old_width = get_dimensions_image_tensor(image)
+    size = _compute_output_size((old_height, old_width), size=size, max_size=max_size)
+    new_height, new_width = size
```

Comment:
Nit: why not directly assign it to the vars?

Reply:
Basically, size is reused below in `_FT.resize(..., size=size, ...)` but yes, we can remove one line and use `[new_height, new_width]` for `_FT.resize`. Thanks!